test/views/ReportingFrequencyViewSpec.scala

```diff
@@ -89,651 +128,538 @@ class ReportingFrequencyViewSpec extends TestSupport {
 
-          pageDocument.select(bullet(2)).text() shouldBe optInGenericContent
+        testContentByIds(pageDocument, newContentEnabled = false)
 
-          pageDocument.select(bullet(2)).attr("href") shouldBe beforeYouStartUrl(isAgentFlag)
-        }
+        pageDocument.getElementById("ceased-business-warning").text() shouldBe "Warning There are currently no businesses on this account. You can add a sole trader or property business on the all businesses page."
 
+        pageDocument.getElementById("ceased-business-link").attr("href") shouldBe "/report-quarterly/income-and-expenses/view/manage-your-businesses"
       }
+      "(new content) return the correct content when the user has a ceased business" in {
+        val isAgentFlag = false
+
+        val reportingFrequencyViewModel: ReportingFrequencyViewModel =
+          ReportingFrequencyViewModel(
+            isAgent = isAgentFlag,
+            Some(optOutChooseTaxYearUrl(isAgentFlag)),
+            Seq(TaxYear(2024, 2025)),
+            Seq(TaxYear(2024, 2025)),
+            Seq(("2024 to 2025", Some("Yes"), Some("Quarterly"))),
+            isAnyOfBusinessLatent = false,
+            displayCeasedBusinessWarning = true
+          )
 
-      "the user is Non-Agent" should {
-
-        "return the correct content when opt in and opt out has multiple tax years" in {
-
-          val isAgentFlag = false
-
-          val reportingFrequencyViewModel: ReportingFrequencyViewModel =
-            ReportingFrequencyViewModel(
-              isAgent = isAgentFlag,
-              Some(confirmOptOutUrl(isAgentFlag)),
-              Seq(TaxYear(2024, 2025), TaxYear(2025, 2026)),
-              Seq(TaxYear(2024, 2025), TaxYear(2025, 2026)),
-              Seq("2024 to 2025" -> Some("Quarterly")),
-              isAnyOfBusinessLatent = false,
-              displayCeasedBusinessWarning = false
-            )
-
-          val pageDocument: Document =
-            Jsoup.parse(
-              contentAsString(
-                view.apply(
-                  viewModel = reportingFrequencyViewModel,
-                  optInOptOutContentUpdateR17 = false
-                )
+        val pageDocument: Document =
+          Jsoup.parse(
+            contentAsString(
+              view.apply(
+                viewModel = reportingFrequencyViewModel,
+                optInOptOutContentUpdateR17IsEnabled = true,
+                nextUpdatesLink = nextUpdatesUrl(isAgentFlag)
               )
             )
+          )
 
-          pageDocument.title() shouldBe title
-
-          testContentByIds(pageDocument)
-
-          pageDocument.select(Selectors.govGuidance).attr("href") shouldBe govGuidanceUrl
-
-          pageDocument.select(bullet(1)).text() shouldBe optOutGenericContent
-
-          pageDocument.select(bullet(1)).attr("href") shouldBe confirmOptOutUrl(isAgentFlag)
+        pageDocument.title() shouldBe title(isNewContentEnabled = true)
```

Comment:
i feel instead of adding logic in to swap between title content we can just have another val and call it here instead.